app/mailers/user_mailer.rb

```diff
@@ -156,7 +159,7 @@ def other_submission_rejected(submission:, user:, artist:)
             unique_args: {
               submission_id: submission.id
             }
-    mail(to: user.email, subject: 'An update about your submission')
+    mail(to: submission.email, subject: 'An update about your submission')
   end
 
   def offer(offer:, artist:, user:)
```

Comment:
**suggestion:** Can we remove the `user` parameter from this method? `OfferService` passes the user associated with an offer's submission to this method, which we can get from the `offer` parameter.

https://github.com/artsy/convection/blob/dd1a2571530faaca4a6d0ed50cb9bb52bcfd7b0a/app/services/offer_service.rb#L152